python-package/xgboost/sklearn.py

```diff
@@ -1262,15 +1263,23 @@ def predict_proba(
             a numpy array of shape array-like of shape (n_samples, n_classes) with the
             probability of each data example being of a given class.
         """
+        # custom obj:      Do nothing as we don't know what to do.
+        # softprob:        Do nothing, output is proba.
+        # softmax:         Use output margin to remove the argmax in PredTransform.
+        # binary:logistic: Expand the prob vector into 2-class matrix after predict.
+        # binary:logitraw: Unsupported, let's deprecate this objective when possible.
```

Comment:
I've seen at least a few users using this objective. Should we actually deprecate it?

Reply:
It's not clear to me in what scenario is it useful.

Reply:
If you want to have an untransformed prediction, use `output_margin` instead.

Reply:
So far we've avoided breaking existing model files, i.e. you could load a very old model file and run predictions. Removing support for `binary:logitraw` will make old models files inaccessible. We don't yet have a formal policy for changes affecting old model files. (e.g. Wait 3 major versions?)

Reply:
I removed the note.